### PR TITLE
bugfix/fix-container

### PR DESCRIPTION
**Pull request by Konmly**
Adaptive 1540px.
1540px

Adaptive 1140px.
1140px_

Adaptive 1024px.
1024px

Adaptive 768px.
768px

Adaptive 600px.
600px

Adaptive 374px.
374px

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
           <h1 class="octopus__title">Обратный звонок на сайт</h1>
           <h3 class="octopus__subtitle">
             Получайте в 3 раза больше звонков сразу после установки виджета на Ваш сайт Сервис
-            окупает свой годовой тариф в первую же неделю
+            окупает свой годовой тариф в первую же неделю
```
Comment (PahaTonk) [inline src/index.html:31]: Is that changes needed?

Comment (Konmly) [reply src/index.html:31]: no

Comment (PahaTonk) [reply src/index.html]: delete it)

**Review by PahaTonk — APPROVED** [review]
